src/backend/storage/file/buffile.c

```diff
@@ -1292,6 +1308,10 @@ BufFileStartCompression(BufFile *file)
 	ResourceOwner oldowner;
 	size_t ret;
 
+	ZSTD_customMem zstdCustomMem;
+	zstdCustomMem.customAlloc = zstdCustomAlloc;
```

Comment:
In the original commit here and below, the new variable declaration was closer to the other declarations.